AlfrescoApp/View Controllers/Take Continous Photos View Controller/CameraController.swift

```diff
@@ -240,3 +246,52 @@ extension CameraController {
         case rear
     }
 }
+
+//MARK: - UIInterfaceOrientation Extension
+
+extension UIInterfaceOrientation {
+    func imageOrientation() -> UIImage.Orientation {
+        switch self {
+        case .portrait:
+            return UIImage.Orientation.right
+        case .portraitUpsideDown:
+            return UIImage.Orientation.left
+        case .landscapeLeft:
+            return UIImage.Orientation.down
+        case .landscapeRight:
+            return UIImage.Orientation.up
+         default:
+            return UIImage.Orientation.up
+        }
+    }
+    
+    func cameraOrientation() -> AVCaptureVideoOrientation {
+        switch self {
+        case .landscapeLeft:
+            return AVCaptureVideoOrientation.landscapeLeft
+        case .landscapeRight:
+            return AVCaptureVideoOrientation.landscapeRight
+        case .portrait:
+            return AVCaptureVideoOrientation.portrait
+        case .portraitUpsideDown:
+            return AVCaptureVideoOrientation.portraitUpsideDown
+        default:
+            return AVCaptureVideoOrientation.portrait
+        }
+    }
+    
+    func imagePropertyOrientation() -> Int {
+        switch self {
+        case .portrait:
+            return 6
```

Comment:
Instead of returning raw values perhaps use an enum with the rawValue property would be more readable